workspaces/copilot/plugins/copilot/src/hooks/useMetrics.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 The Backstage Authors
```

Comment:
Can you look all these over, some say 2024 but others don't match like this one. I would expect them all to be 2024